Reformat outreach chapter, add images, update toc and open for review (#1709)
alan-turing-institute/the-turing-way: book/website/communication/outreach.md

(cm-outreach)=
# Outreach in Data Science and Research

> Outreach is an umbrella term for a broad range of activities which engage researchers with audiences external to the core research team, including other scientists, the general public, journalists, investors and policy makers.

Author: ```suggestion
Outreach is an umbrella term for a broad range of activities that engage researchers with audiences external to the core research team, including other scientists, the general public, journalists, investors and policymakers.
```
